pyangbind/lib/pybindJSON.py

```diff
@@ -17,11 +17,19 @@
 See the License for the specific language governing permissions and
 limitations under the License.
 """
-from .serialise import pybindJSONEncoder, pybindJSONDecoder, pybindJSONIOError
-from .serialise import pybindIETFJSONEncoder
+from __future__ import unicode_literals
+
+from pyangbind.lib.serialise import pybindJSONEncoder, pybindJSONDecoder, pybindJSONIOError
+from pyangbind.lib.serialise import pybindIETFJSONEncoder
 import json
 import copy
 
+# Kludge to temporarily fix unicode references
+try:
+  unicode
+except NameError:
+  unicode = str
+
```

Comment:
Can you explain why this change is required? If we have unicode literals above, shouldn't we just convert over everything to being unicode literals and remove the references to `unicode`?